shared/core/src/main/scala/org/hyperledger/identus/shared/models/KeyId.scala

```diff
@@ -0,0 +1,6 @@
+package org.hyperledger.identus.shared.models
+
+opaque type KeyId = String
```

Comment:
@mineme0110, I use kid (KeyId) in the scope of OIDC work. 
By the OID4VCI specification, `kid` is a DIDUrl, so I made a corresponding logic to parse it [here](https://github.com/hyperledger/identus-cloud-agent/blob/8bc2018bd1ca6d1aa679e1935cde5996602b8ae5/castor/src/main/scala/org/hyperledger/identus/castor/core/model/did/DIDUrl.scala#L5).
The [reference](https://openid.net/specs/openid-4-verifiable-credential-issuance-1_0.html#name-jwt-proof-type) to the spec says: `If the Credential shall be bound to a DID, the kid refers to a DID URL which identifies a particular key in the DID Document that the Credential shall be bound to.`

Reply:
Yes  I understand but the holder can have mutliple keys and this is used when you accept offer which to kid to use

Reply:
Also , We are currently missing the "kid" in the JWT header. As the issuer, when you sign the credential, the "kid" in the header should indicate which key was used to sign it.

We need to create a story to address this for both VCJWT (the work was done in a previous PR) and SDJWT. However, the underlying Rust library does not support this feature yet. I'm uncertain whether we should make the change now or switch to a new library and make the changes later for the issuer.

Reply:
it's true.